### ISSUE TITLE
Freezer locations not recording accurately

### ISSUE THREAD

**Issue by BlakeDraper** [USGS-WiM/liliservices#74]
I have the client now set up to query for the last occupied spot and display that to the user so they can decide where to start the set of aliquots they are freezing/creating. To test this, I did the thing the user would presumably do - last occupied spot is reported as 3, so I filled in spot 4 as the starting point for the 3 new aliquots being created with my new aliquot POST. 

Request succeeds and aliquots records are created, but a check on the aliquots table and the freezer locations table indicates that the 3 new records just created start at spot 1 (see image below). I have done this twice now, and those submissions combined with the first submission (total of 3) mean that we have 3 samples, with 3 aliquots each, starting at `Rack 1, Box 1, Row 1, Spot 1` and going up through `Rack 1, Box 1, Row 1, Spot 3`.  

In my second and third submissions, I indicated spot 4 as the starting point (based on last occupied spot as reported by the web services), but nonetheless spot 1 was recorded for both of those. 

TL;DR: services business logic seems to be replacing spot number submission with `1`.

![image](https://user-images.githubusercontent.com/6978380/35476648-f3c0f950-0378-11e8-9d95-2157a43fe338.png)
